ethereum/contracts/Starport.sol

```diff
@@ -31,7 +31,6 @@ contract Starport {
     event Unlock(address indexed account, uint amount, address asset);
     event UnlockCash(address indexed account, uint amount, uint128 principal);
     event ChangeAuthorities(address[] newAuthorities);
-    event SetFutureYield(uint128 nextCashYield, uint128 nextCashYieldIndex, uint nextCashYieldStart);
```

Comment:
I'm not sure we need to remove this. I think it's fine if both the Starport and Cash Token emit such an event.